README.md

```diff
@@ -78,6 +78,66 @@ Currently, the client id and domain of an existing Cognito user pool programmati
 
 # Gitflow Model
 [VEDA pipeline gitflow](./GITFLOW.md)
+
+# Ingestion Pipeline Overview
+
+This pipeline is designed to handle the ingestion of both vector and raster data. The ingestion can be performed using the `veda-discover` DAG. Below are examples of configurations for both vector and raster data.
+
+## Ingestion Configuration
+
+### Vector Data Ingestion
+```json
+{
+  "collection": "",
+  "bucket": "",
+  "prefix": "",
+  "filename_regex": ".*.csv$",
+  "id_template": "-{}",
+  "datetime_range": "",
+  "vector": true,
+  "x_possible": "longitude",
+  "y_possible": "latitude",
+  "source_projection": "EPSG:4326",
+  "target_projection": "EPSG:4326",
+  "extra_flags": ["-overwrite", "-lco", "OVERWRITE=YES"]
+}
+```
+
+### Raster Data Ingestion 
+```json
+{
+    "collection": "",
+    "bucket": "",
+    "prefix": "",
+    "filename_regex": ".*.tif$",
+    "datetime_range": "",
+    "assets": {
+        "co2": {
+            "title": "",
+            "description": ".",
+            "regex": ".*.tif$"
+        }
+    },
+    "id_regex": ".*_(.*).tif$",
+    "id_template": "-{}"
+}
+
+```
+## Configuration Fields Description
+- `collection`: The collection_id of the raster or vector data.
+- `bucket`: The name of the S3 bucket where the data is stored.
+- `prefix`: The location within the bucket where the files are to be discovered.
+- `filename_regex`: A regex expression used to filter files based on naming patterns.
+- `id_template`: The format used to create item identifiers in the system.
+- `vector`: Set to true to trigger the generic vector ingestion pipeline.
+- `vector_eis`: Set to true to trigger the vector ingestion pipeline.
```

Comment:
Suggestion: Can this description say "Set to true to trigger the **EIS specific** vector ingestion pipeline" 

The way it's written now it's difficult to find the difference between this one and the generic vector